src/ui/public/route_based_notifier/__tests__/index.js

```diff
@@ -10,12 +10,15 @@ describe('ui/route_based_notifier', function () {
 
```

Comment:
Clearing the notifier before each test doesn't clean up the changes you've made. It needs to happen in the after each handler, right?